shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/controllers/BridgeExecutorController.java

```diff
@@ -91,6 +92,11 @@
             // TODO: notify the manager if in FailureState: .status.Type = Ready and .status.Reason = DeploymentFailed
```

Comment:
@kostola Can you create a JIRA issue for us to do this please. I don't think we have one.

Reply:
Honestly I have no idea what this TODO is for...it was added by @ricardozanini

Reply:
I will ask him why he put this TODO here and open a task accordingly :+1:

Reply:
Issue created: https://issues.redhat.com/browse/MGDOBR-320